js/libs/core.js

```diff
@@ -86,7 +86,11 @@ Fliplet.FormBuilder = (function() {
             $($vm.$el).removeClass('has-error');
           }
         }
-      }
+      };
+
+      component.methods.saveProgress = function($event) {
+        this.$emit('_input', this.name, $event.target.value);
```

Comment:
@squallstar This isn't debounced, while the user could be typing very quickly, firing this many times. Do you see a problem with it?

Reply:
I agree with debouncing this to something like 200ms. @tonytlwu confirm a good debounce time in regards to average typing speed :-)

Reply:
Fixed